ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Query.hs

```diff
@@ -60,6 +62,20 @@ import           Ouroboros.Network.Block (MaxSlotNo, maxSlotNoFromWithOrigin)
 -- In the latter case, we don't take blocks already in the ImmutableDB into
 -- account, as we know they /must/ have been \"immutable\" at some point, and,
 -- therefore, /must/ still be \"immutable\".
+getCurrentChain'
```

Comment:
Minor: If you defined `getCurrentChain'` last, then would the diff be shorter/better aligned?

Reply:
Right, that seems to be the case, will change :+1: